android/app/src/main/kotlin/com/yubico/authenticator/MainActivity.kt

```diff
@@ -801,4 +805,7 @@ class MainActivity : FlutterFragmentActivity() {
     }
 
     private fun isPortraitOnly() = resources.getBoolean(R.bool.portrait_only)
+
+    private val defaultLogLevel =
+        if (BuildConfig.DEBUG) Log.LogLevel.DEBUG else Log.LogLevel.INFO
```

Comment:
This was already happening before, right? Is that set in some other place, and can that now be removed?